### PR TITLE
meshcentral: 0.8.87 -> 0.8.98

### PR DESCRIPTION
**Pull request by lheckemann**
###### Things done

- [x] Deployed to my meshcentral server and tested that it still works
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 133086` at a443496e run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>meshcentral</li>
  </ul>
</details>
<details>
  <summary>4 suggestions:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `jq`.
Near pkgs/tools/admin/meshcentral/default.nix:1:56:
```
  |
1 | { lib, fetchpatch, fetchzip, yarn2nix-moretea, nodejs, jq, dos2unix }:
  |                                                        ^
```
</li>
    <li>warning: unused-argument

Unused argument: `fetchpatch`.
Near pkgs/tools/admin/meshcentral/default.nix:1:8:
```
  |
1 | { lib, fetchpatch, fetchzip, yarn2nix-moretea, nodejs, jq, dos2unix }:
  |        ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md">build-tools-in-build-inputs</a>

rsync is a build tool so it likely goes to `nativeBuildInputs`, not `buildInputs`.

Near pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix:294:7:
```
    |
294 |       buildInputs = [ yarn nodejs rsync ] ++ extraBuildInputs;
    |       ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md">build-tools-in-build-inputs</a>

yarn is a build tool so it likely goes to `nativeBuildInputs`, not `buildInputs`.

Near pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix:294:7:
```
    |
294 |       buildInputs = [ yarn nodejs rsync ] ++ extraBuildInputs;
    |       ^
```
</li>
  </ul>
</details>

File: pkgs/tools/admin/meshcentral/default.nix [hunk 1]
```diff
@@ -1,10 +1,10 @@
 { lib, fetchpatch, fetchzip, yarn2nix-moretea, nodejs, jq, dos2unix }:
 yarn2nix-moretea.mkYarnPackage rec {
-  version = "0.8.87";
+  version = "0.8.98";
 
   src = fetchzip {
-    url = "https://registry.npmjs.org/meshcentral/-/meshcentral-0.8.87.tgz";
-    sha256 = "1jb65pvbld83mdjdb4f4z2brqsdh3b1mvnjdhbllcsn35m705cp5";
+    url = "https://registry.npmjs.org/meshcentral/-/meshcentral-0.8.98.tgz";
```
Comment (SuperSandro2000) [inline pkgs/tools/admin/meshcentral/default.nix:6]:
```suggestion
    url = "https://registry.npmjs.org/meshcentral/-/meshcentral-${version}.tgz";
```

Comment (lheckemann) [reply pkgs/tools/admin/meshcentral/default.nix:6]: The update script inserts the version like this.

Comment (jansol) [reply pkgs/tools/admin/meshcentral/default.nix:6]: Then it would be prudent to change the update script to not repeat itself unnecessarily as well.

Comment (elseym) [reply pkgs/tools/admin/meshcentral/default.nix:6]: valid point, but out of scope for this pr. this is just a minor version bump.

if you want, you could have a look at the update-script... i think `pkgs/common-updater/scripts/update-source-version` would be a good place to start. :slightly_smiling_face:

Comment (SuperSandro2000) [reply pkgs/tools/admin/meshcentral/default.nix:6]: That is not even a custom updater which makes this change even more mundane.

**Pull request merged**